src/Console/Command/BaseAspectCommand.php

```diff
@@ -37,16 +37,26 @@ protected function configure()
     }
 
     /**
-     * {@inheritDoc}
+     * Loads aspect kernel.
+     *
+     * Aspect kernel is loaded by executing loader and fetching singleton instance.
+     * If your application environment initializes aspect kernel differently, you may
+     * modify this metod to get aspect kernel suitable to your needs.
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function loadAspectKernel(InputInterface $input, OutputInterface $output)
     {
         $loader = $input->getArgument('loader');
         $path   = stream_resolve_include_path($loader);
         if (!is_readable($path)) {
             throw new \InvalidArgumentException("Invalid loader path: {$loader}");
         }
+
+        ob_start();
```

Comment:
When $path is included, it's being executed. Usually, that is some front controller of web application and homepage is rendered, filling console with bunch of HTML code.

Surrounding it with  ob_start() and ob_clean() will just keep console nice and clean, making it more user friendly.